src/book.py

```diff
@@ -136,19 +163,34 @@ def convert_structure(structure):
         for chapter in chapters:
             for line in chapter.split("\n"):
                 if 'paragraphs' in line.lower():
-                    chapter_information.append({'title': line.split('): ')[1], 'paragraphs': []})
-                elif 'paragraph' in line.lower():
+                    chapter_information.append(
+                      {'title': line.split('): ')[1], 'paragraphs': []}
+                                              )
+                if 'paragraph' in line.lower():
                     chapter_information[-1]['paragraphs'].append(
-                        {'title': line.split('): ')[1], 'words': line.split('(')[1].split(')')[0].split(' ')[0]})
-            chapter_information[-1]['paragraph_amount'] = len(chapter_information[-1]['paragraphs'])
-
+                    {'title': line.split('): ')[1], 'words': line.split(
+                      '(')[1].split(')'
+                                                                       )[0].split(
+                      ' '
+                                                                                 )[0]}
+                                                                )
+                elif chapter_information == []:
+                  ## I can't figure out the code for this
```

Comment:
Sounds unfinished, does it work?

Reply:
Yes this part of the code works so far I just formatted it differently